src/Tasks/Microsoft.Common.CurrentVersion.targets

```diff
@@ -281,6 +281,8 @@ Copyright (C) Microsoft Corporation. All rights reserved.
     <!-- Example, c:\MyProjects\MyProject\bin\debug\MyAssembly.dll -->
     <TargetPath Condition=" '$(TargetPath)' == '' ">$(TargetDir)$(TargetFileName)</TargetPath>
 
+    <TargetRefPath Condition=" '$(TargetRefPath)' == '' and '$(Deterministic)' == 'true' ">$([System.IO.Path]::Combine(`$([System.IO.Path]::GetDirectoryName($([System.IO.Path]::GetFullPath(`$(TargetPath)`))))`, 'ref', `$(TargetFileName)`))</TargetRefPath>
```

Comment:
That was my original plan, but it would break the case where `TargetPath` is explicitly specified, which seems to be legal from my reading of the prior set of conditions.

I'd be fairly ok with saying "if you want to do that odd thing, you must also specify this" but it needs to be an explicit choice.